src/linux/backend-hid.cpp

```diff
@@ -822,10 +837,14 @@ namespace librealsense
             // get the specific name of sampling_frequency
             _sampling_frequency_name = get_sampling_frequency_name();
 
+            // get the specific name of sensitivity
+            _sensitivity_name = get_sensitivity_name();
```

Comment:
What kind of names can be set here?

Reply:
This gives you the name of the file to write to when setting the gyro sensitivity values, similar implementation like the frequency values

Reply:
OK, got it thanks